doc/en/user/source/configuration/status.rst

```diff
@@ -209,98 +236,26 @@ The available system information is:
-
-Enable All Statistics
-^^^^^^^^^^^^^^^^^^^^^
-Enables the displays the information of the listed System Status Info properties. It is disabled by default. Disabling is useful if GeoServer is generating a high CPU load due to system status computations.
-
-REST interface
-^^^^^^^^^^^^^^
-
-It is possible to request the available system information (monitoring data) through GeoServer REST API. The supported formats are XML, JSON and HTML. 
+JVM Console
+-----------
 
-The available REST endpoints are: ::
+For information on the live Java Runtime Environment the :guilabel:`JVM Console` tab provides access to two useful troubleshooting tools.
 
-    /geoserver/rest/about/system-status
-    
-    /geoserver/rest/about/system-status.json
-
-    /geoserver/rest/about/system-status.xml
-
-    /geoserver/rest/about/system-status.html
-
-The HTML representation of the system data is equal to the ``System status`` tab representation:
-
- .. figure:: img/resthtml.png
-   :align: center
-
-|
-
-The XML and JSON representations are quite similar, for each system information the following attributes will be available:
-
-.. list-table::
-   :widths: 40 60
-
-   * - **Name**
-     - **Description**
-   * - name
-     - name of the metric
-   * - available
-     - TRUE if the system information value is available
-   * - description
-     - description of this system information
-   * - unit
-     - unit of the system information, can be empty 
-   * - category
-     - category of this system information
-   * - priority
-     - this value can be used to render the metrics in a predefined order
-   * - identifier
-     - identifies the resource associated with the metric, e.g. file partition name
-
-Example of XML representation: ::
-
-	<metrics>
-    <metric>
-      <value>99614720</value>
-      <available>true</available>
-      <description>Partition [/dev/nvme0n1p2] total space</description>
-      <name>PARTITION_TOTAL</name>
-      <unit>bytes</unit>
-      <category>FILE_SYSTEM</category>
-      <identifier>/dev/nvme0n1p2</identifier>
-      <priority>507</priority>
-    </metric>
-		(...)
-
-Example of JSON representation: ::
-
-	{
-	    "metrics": {
-	        "metric": [
-	            {
-                  "available": true,
-                  "category": "FILE_SYSTEM",
-                  "description": "Partition [/dev/nvme0n1p2] total space",
-                  "identifier": "/dev/nvme0n1p2",
-                  "name": "PARTITION_TOTAL",
-                  "priority": 507,
-                  "unit": "bytes",
-                  "value": 99614720
-              },
-	            (...)
+Press :guilabel:`Thread Dump` for a summary of all active threads. This is primarily used to troubleshoot performance issues and a non-responsive system. This can be used to identify when significant works is happening in the background, or if threads are stuck waiting on a resource.
```

Comment:
```suggestion
Press :guilabel:`Thread Dump` for a summary of all active threads. This is primarily used to troubleshoot performance issues and a non-responsive system. This can be used to identify when significant work is happening in the background, or if threads are stuck waiting on a resource.
```